python/ray/_private/state.py

```diff
@@ -81,7 +81,10 @@ def _really_init_global_state(self):
         self.global_state_accessor.connect()
 
     def actor_table(
-        self, actor_id: str, job_id: ray.JobID = None, actor_state_name: str = None
```

Comment:
Actually for internal APIs, it's probably better to accept JobID since it's strongly typed: you avoid the problem of accidentally passing in a binary string to a method that accepts hex string.

Reply:
makes sense. update